packages/remix-netlify/package.json

```
@@ -15,9 +15,9 @@
     "@remix-run/node": "1.3.2"
   },
   "peerDependencies": {
-    "@netlify/functions": "^0.10.0"
+    "@netlify/functions": "^0.10.0 || ^0.11.0 || ^1.0.0"
   },
   "devDependencies": {
-    "@netlify/functions": "^0.10.0"
+    "@netlify/functions": "^0.10.0 || ^0.11.0 || ^1.0.0"
```

```suggestion
    "@netlify/functions": "^1.0.0"
```

The devDependency should be the latest version